syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;
import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetAttributeBucketInDiseaseRequest {
    uint32 disease_id = 1;
    uint32 symptom_id = 2;
    uint32 attribute_id = 3;
}

message GetAttributeBucketInDiseaseResponse {
	uint32 bucket = 1;
    jio.brain.proto.knowledge.healthcare.base.Status status = 2;
}